install/kubernetes/cilium/charts/agent/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+You have successfully installed Cilium {{ .Chart.name }}
+
+Your release version is {{ .Chart.version }}
+
+Join the Cilium Slack channel to chat with Cilium developers and other Cilium users.
+This is a good place to learn about Cilium, ask questions, and share your experiences.
```

Comment:
Might be worth linking here to the Cilium docs page that has the links to Slack.